daemon/config_unix.go

```diff
@@ -120,3 +121,13 @@ func (config *Config) GetAllRuntimes() map[string]types.Runtime {
 	config.reloadLock.Unlock()
 	return rts
 }
+
+func (config *Config) isSwarmCompatible() error {
+	if config.IsValueSet("cluster-store") || config.IsValueSet("cluster-advertise") {
```

Comment:
Why did you change this code from `config.ClusterStore != "" ` to `config.IsValueSet("cluster-store")`?
As I know, `config.IsValueSet()` checks only configs which were set in the **configuration file**.
So this code always returns `false` even if using `--cluster-store` option.